log/file.go

```diff
@@ -44,15 +44,19 @@ type fileHook struct {
 	w              io.WriteCloser
 	bw             *bufio.Writer
 	levels         []logrus.Level
+	done           chan struct{}
 }
 
 // FileHookFromConfigLine returns new fileHook hook.
 func FileHookFromConfigLine(
-	ctx context.Context, fallbackLogger logrus.FieldLogger, line string,
+	ctx context.Context, fallbackLogger logrus.FieldLogger, line string, done chan struct{},
 ) (logrus.Hook, error) {
+	// TODO: fix this so it works correctly with relative paths from the CWD
```

Comment:
what do you mean by this?

Reply:
Noticed that we don't explicitly check if we got a relative path here and instead just open exactly what we were given: https://github.com/grafana/k6/blob/1e28a3e1c8655d2118c823113e7a90ce44736cfe/log/file.go#L108

This works, for now, since we're directly using the `os` package and it will properly handle relative paths based on the current working directory. However, after we switch to `afero.Fs` here, we will need to explicitly account for it.

Reply:
can you expand this comment there with that info :)

Reply:
No need, I plan to fix it with a small commit in https://github.com/grafana/k6/pull/2412 :sweat_smile:

Reply:
FWIW, this is the fix and the test to verify it: https://github.com/grafana/k6/pull/2412/commits/7574bd2c2488a7f6a3bc804a8f12aede47383b17